niftynet/layer/loss_classification_multi.py

```diff
@@ -0,0 +1,265 @@
+                    tuple)):
+                pred_multi = [pred_multi]
+                if self._num_classes > 0 and pred_multi is not None:
+
+                    pred_multi = [tf.reshape(pred, [-1,
+                                                self._num_raters,
+                                                self._num_classes])
+                                    for pred in pred_multi]
+
+
+            data_loss = []
+            if ground_truth is not None:
+                if pred_multi is not None:
+                    if pred_ave is not None:
+                        for pred, pred_mul in zip(pred_ave, pred_multi):
+                            if self._loss_func_params:
+                                data_loss.append(
+                                    self._data_loss_func(ground_truth,
+                                                         pred, pred_mul,
+                                                         **self._loss_func_params))
+                            else:
+                                data_loss.append(self._data_loss_func(
+                                    ground_truth, pred, pred_mul))
+                    else:
+                        for pred_mul in pred_multi:
+                            if self._loss_func_params:
+                                data_loss.append(
+                                    self._data_loss_func(ground_truth,
+                                                         pred_mul,
+                                                         **self._loss_func_params))
+                            else:
+                                data_loss.append(self._data_loss_func(
+                                    ground_truth, pred_mul))
+                else:
+                    for pred in pred_ave:
+
+                        if self._loss_func_params:
+                            data_loss.append(self._data_loss_func(
+                                pred, ground_truth,
+                                **self._loss_func_params))
+                        else:
+                            data_loss.append(self._data_loss_func(
+                                pred, ground_truth))
+            elif pred_multi is not None:
+                for pred, pred_mul in zip(pred_ave, pred_multi):
+                    if self._loss_func_params:
+                        data_loss.append(self._data_loss_func(
+                            pred, pred_mul,
+                            **self._loss_func_params))
+                    else:
+                        data_loss.append(self._data_loss_func(
+                            pred, pred_mul))
+
+            if weight_batch is not None:
+                return tf.reduce_mean(weight_batch/tf.reduce_sum(
+                    weight_batch) * data_loss[0])
+            else:
+                return tf.reduce_mean(data_loss)
+
+
+def labels_to_one_hot(ground_truth, num_classes=1, nrater=6):
```

Comment:
`nrater` is not used in this function, perhaps we could `from niftynet.layer.loss_segmentation import labels_to_one_hot` instead of creating a new copy here?

Reply:
The second output (output_shape) is required and not present in the function of segmentation loss. For now I have left but am happy to change loss_segmentation instead and clall it from there